app/src/main/java/xyz/kotlout/kotlout/view/ProfileActivity.java

```diff
@@ -161,6 +161,8 @@ public boolean onOptionsItemSelected(@NonNull MenuItem item) {
 
         if (UserController.validateEmail(newEmail) == true
             && UserController.validatePhoneNumber(newPhone) == true) {
+          Toast.makeText(this, "Valid Entry", Toast.LENGTH_SHORT);
```

Comment:
I don't think the toast displays unless you call the .show() method, is this intentional?

Reply:
Toasts were from when I was doing tests, they need to be removed anyways.